src/components/Comment.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+function Comment({ comment }) {
```

Comment:
Why do you need to pass the whole object as a prop? will be better to pass name and body separately